packages/clay-dataset-display/demos/index.html

```diff
@@ -0,0 +1,418 @@
+						difficulty: 6,
+						downloadHref: '#',
+						format: 'PDF',
+						id: '6',
+						name: 'Pizza',
+						origin: 'Italy',
+						spicyLevel: ['No Spicy'],
+						time: '30 min.',
+					},
+					{
+						difficulty: 5,
+						downloadHref: '#',
+						format: 'PDF',
+						id: '7',
+						image: './carcamusas.jpg',
+						name: 'Carcamusas',
+						origin: 'Toledo',
+						spicyLevel: ['Spicy'],
+						time: '50 min.',
+					},
+					{
+						difficulty: 5,
+						downloadHref: '#',
+						format: 'PDF',
+						id: '8',
+						name: 'Tacos',
+						origin: 'Mexico',
+						spicyLevel: ['Very Spicy'],
+						time: '30 min.',
+					},
+				],
+				label: "Recipes",
+			},
+			{
+				items: [
+					{
+						id: '9',
+						initials: 'FA',
+						name: 'Ferrán Adriá',
+						style: 'fusion',
+						type: 'chef',
+					},
+					{
+						id: '10',
+						image: './ramsay.jpg',
+						initials: 'GR',
+						name: 'Gordon Ramsay',
+						style: 'general',
+						type: 'chef',
+					},
+					{
+						id: '11',
+						initials: 'AC',
+						name: 'Alberto Chicote',
+						style: 'general',
+						type: 'chef',
+					},
+				],
+				label: 'Chefs',
+			}
+		];
+
+		var spritemap = '../../../node_modules/clay/build/images/icons/icons.svg';
+
+		items.forEach(item => {
+			item.items.forEach(nestedItem => {
+				nestedItem.actionItems = actionItemsWithQuickItems;
+			});
+		});
+
+		new metal.ClayDatasetDisplay({
+			actionItems: actionItemsWithQuickItems,
+			filterItems: [
+				{
+					items: [
+						{
+							label: 'Item 1',
+							inputName: 'filteritem1checkbox',
+							inputValue: '1',
+							type: 'checkbox',
+						},
+						{
+							checked: true,
+							label: 'Item 2',
+							inputName: 'filteritem2checkbox',
+							inputValue: '2',
+							type: 'checkbox',
+						},
+						{
+							disabled: true,
+							label: 'Item 3',
+							inputName: 'filteritem3checkbox',
+							inputValue: '3',
+							type: 'checkbox',
+						},
+					],
+					label: 'Filter By',
+					type: 'group',
+				},
+				{
+					items: [
+						{
+							label: 'Item 1',
+							inputName: 'orderitem1checkbox',
+							inputValue: '1',
+							type: 'checkbox',
+						},
+						{
+							checked: true,
+							label: 'Item 2',
+							inputName: 'orderitem2checkbox',
+							inputValue: '2',
+							type: 'checkbox',
+						},
+						{
+							disabled: true,
+							label: 'Item 3',
+							inputName: 'orderitem3checkbox',
+							inputValue: '3',
+							type: 'checkbox',
+						},
+					],
+					label: 'Order By',
+					type: 'group',
+				},
+				{
+					items: [
+						{
+							label: 'Item 1',
+							inputValue: '1',
+						},
+						{
+							checked: true,
+							label: 'Item 2',
+							inputValue: '2',
+						},
+						{
+							disabled: true,
+							label: 'Item 3',
+							inputValue: '3',
+						},
+					],
+					inputName: 'item1radio',
+					label: 'Group 1',
+					type: 'radiogroup',
+				}
+			],
+			items: items,
+			plusButton: {
```

Comment:
What does this API refer to in particular? The Management Toolbar add actions? We should find a better name for this item...

Reply:
https://github.com/metal/metal-clay-components/pull/259